### DISCUSSION TITLE
FilterQuality in Flame

### DISCUSSION THREAD

**Discussion by Vasilisk8** [flame-engine/flame#1917] [Q&A] [answered]
Faced with poor rendering quality on the Web. In the process of research, I realized that a bad render is only when using canvaskit (everything is fine in html). In flutter, the problem repeats, but is solved by assigning FilterQuality.high. How to set FilterQuality in Flame?

**Comment by ufrshubham** [comment, marked as answer]
You can supply a custom `Paint` object with desired `filterQuality` to any component that uses `HasPaint` mixin

**Reply by ufrshubham** [reply]
@spydon, you can mark this one is answered

**Comment by Vasilisk8** [comment]
Thanks, everything worked out. I bow to the floor:)
